src/features/gateway/steps/GatewayInitialStep.tsx

```diff
@@ -359,22 +369,51 @@ export const GatewayInitialStep: FunctionComponent<GatewayStepProps> = ({
               usdValue={amountUsd}
               value={amount}
               errorText={
-                showAmountError && hasAmountBalanceError ? (
-                  <span>
-                    {t("gateway.amount-too-big-error")}{" "}
-                    <TooltipWithIcon
-                      title={
-                        <span>{t("gateway.amount-too-big-error-tooltip")}</span>
-                      }
-                    />
-                  </span>
-                ) : (
-                  ""
-                )
+                showAmountError || showMinimumAmountError ?
+                  <>
+                    {showAmountError &&
```

Comment:
it has exactly the same HTML structure, so maybe wrap it in `<MessageWithTooltip massage={} tooltip={} />` for reusability.